measures/ResStockArguments/measure.rb

```diff
@@ -389,6 +389,7 @@ def run(model, runner, user_arguments)
 
     # assign the user inputs to variables
     args = runner.getArgumentValues(arguments(model), user_arguments)
+    args = convert_args(model, args)
```

Comment:
Can't you just pass in the measure arguments like so?

```suggestion
    args = convert_args(arguments(model), args)
```

Reply:
We want only the original arguments from BuildResidentialHPXML, no?

Reply:
I guess I assumed there's no harm on having all the arguments, but maybe there's a nuance I'm missing. Feel free to disregard my comment if you don't think it's better/correct.

Reply:
Maybe you're right. I'll try it.

Reply:
Tried it. One nuance is that `geometry_unit_cfa` gets turned into a string argument. But we can just handle that one separately.

Reply:
Er wait. This doesn't work at all -- `measure(arguments)` changes all BuildResidentialHPXML double and integers arguments to string...

Reply:
Oh, right. Okay then I guess you have to do this.